prime-router/metadata/hl7_mapping/OML_O21/base/service-request-order-observation.yml

```diff
@@ -1,8 +1,27 @@
 constants:
   hl7OBRField: '%{hl7SegmentGroup}/OBR'
 elements:
+  - name: order-observation-set-id
```

Comment:
Curious how much this mapping will differ with an `OBR` in and ORU_R01 message?  Seems like this would be an opportunity to use a common schema here so any fixes only need to be made in one spot.

Reply:
@mkalish That's a good point, we were just discussing that yesterday. One difference is that the resource for `ORU`'s `OBR` is taken from `DiagnosticReport` and for `OML` is taken from `ServiceRequest`. In any case, I'll take a deeper look today to look for opportunities to share a common schema

Reply:
@mkalish I extracted the common fields into this schema `metadata/hl7_mapping/common/observation-request.yml`